css-align-3/content-distribution/place-content-shorthand-003.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+  <head>
+      <title>CSS Boc Alignment: place-content shorthand - initial value</title>
+      <link rel="author" title="Javier Fernandez" href="mailto:jfernandez@igalia.com" />
+      <link rel="help" href="http://www.w3.org/TR/css3-align/#propdef-place-content" />
+      <meta name="assert" content="Check that place-content's initial value expands to 'align-content' and 'justify-content'." />
```

Comment:
The part of the sentence " ... value expands to 'align-content' and 'justify-content'." tried to mean that precisely.